packages/core/src/transport/batch.spec.ts

```diff
@@ -150,4 +150,21 @@ describe('batch', () => {
 
     expect(transport.send).toHaveBeenCalledWith('{"message":"10"}\n{"message":"11"}', jasmine.any(Number), undefined)
   })
+
+  it('should be able to use telemetry in the httpRequest.send', () => {
+    const fakeRequest = {
+      send(data: string) {
+        addTelemetryDebugFake()
+        transport.send(data, BATCH_BYTES_LIMIT)
+      },
+    } as unknown as HttpRequest
+    const batch = new Batch(fakeRequest, 10, 100, 100, 100)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏unless we care about having those exact values for the batch, we could use the global constants

```suggestion
    const batch = new Batch(fakeRequest, BATCH_MESSAGES_LIMIT, BATCH_BYTES_LIMIT, MESSAGE_BYTES_LIMIT, FLUSH_TIMEOUT)
```